src/main/java/com/clussmanproductions/trafficcontrol/gui/GuiProxy.java

```diff
@@ -125,5 +161,8 @@ public Object getClientGuiElement(int ID, EntityPlayer player, World world, int
 		public static final int STREET_SIGN = 6;
 		public static final int TRAFFIC_LIGHT_5_FRAME = 7;
 		public static final int TRAFFIC_LIGHT_DOGHOUSE_FRAME = 8;
+		public static final int TRAFFIC_LIGHT_ONE_FRAME = 9;
```

Comment:
"1" instead of "one."  Do the same below too please

Reply:
Fixed